common/dna_data/wc_nightelf_dna.txt

```diff
@@ -117,4 +126,543 @@ illidan_stormrage_dna = { # by ercarp
+			face_detail_nasolabial={ "nasolabial_01" 172 "nasolabial_03" 65 }
+			face_detail_nose_ridge_def={ "nose_ridge_def_neg" 72 "nose_ridge_def_neg" 250 }
+			face_detail_nose_tip_def={ "nose_tip_def" 221 "nose_tip_def" 98 }
+			face_detail_temple_def={ "temple_def" 174 "temple_def" 174 }
+			expression_brow_wrinkles={ "brow_wrinkles_03" 220 "brow_wrinkles_03" 66 }
+			expression_eye_wrinkles={ "eye_wrinkles_02" 218 "eye_wrinkles_01" 143 }
+			expression_forehead_wrinkles={ "forehead_wrinkles_03" 127 "forehead_wrinkles_03" 55 }
+			expression_other={ "cheek_wrinkles_both_01" 0 "cheek_wrinkles_both_01" 54 }
+			complexion={ "complexion_5" 198 "complexion_5" 192 }
+			gene_height={ "normal_height" 176 "normal_height" 176 }
+			gene_bs_body_type={ "body_fat_head_fat_low" 95 "body_fat_head_fat_low" 119 }
+			gene_bs_body_shape={ "body_shape_triangle_full" 255 "body_shape_triangle_full" 15 }
+			gene_bs_bust={ "bust_clothes" 93 "bust_shape_3_half" 93 }
+			gene_age={ "old_4" 23 "old_2" 23 }
+			gene_eyebrows_shape={ "no_eyebrows" 173 "no_eyebrows" 173 }
+			gene_eyebrows_fullness={ "no_eyebrows" 0 "no_eyebrows" 70 }
+			gene_body_hair={ "body_hair_sparse" 0 "body_hair_sparse" 0 }
+			gene_hair_type={ "hair_straight" 127 "hair_straight" 127 }
+			gene_baldness={ "no_baldness" 127 "no_baldness" 127 }
+			eye_accessory={ "normal_eyes" 153 "normal_eyes" 153 }
+			teeth_accessory={ "normal_teeth" 0 "normal_teeth" 0 }
+			eyelashes_accessory={ "normal_eyelashes" 99 "normal_eyelashes" 99 }
+			gene_bs_ear_lenght={ "floppy_ear_length" 238 "floppy_ear_length" 254 }
+			special_eyebrows={ "night_elven_eyebrows" 255 "night_elven_eyebrows" 57 }
+			special_eyes={ "magic_no_pupils_eyes" 168 "magic_eyes" 168 }
+			gene_bs_mouth_tusks={ "no_mouth_tusks" 0 "mouth_fangs" 0 }
+			static_tusks={ "no_static_tusks" 0 "no_static_tusks" 0 }
+			gene_bs_ear_tip={ "ear_tip_pos" 255 "ear_tip_pos" 0 }
+			gene_bs_fingers={ "fingers_troll" 0 "fingers_troll" 0 }
+			gene_skin_value={ "skin_value_neg" 31 "skin_value_neg" 127 }
+			gene_race={ "creature_human" 127 "creature_human" 127 }
+			tendrils={ "no_tendrils" 0 "no_tendrils" 0 }
+			gene_facial_markings={ "malfurion_markings" 127 "no_markings" 127 }
+			horns={ "no_horns" 0 "no_horns" 0 }
+			gene_sexual_dimorphism={ "no_dimorphism" 127 "no_dimorphism" 127 }
+			gene_grade={ "plantigrade" 127 "plantigrade" 127 }
+			tail={ "no_tail" 0 "no_tail" 0 }
+			hooves={ "no_hoves" 0 "no_hoves" 0 }
+			eye_left_accessory={ "normal_eyes" 0 "normal_eyes" 0 }
+			gene_body_markings={ "no_body_markings" 127 "no_body_markings" 127 }
+			gene_mustaches={ "mustaches" 167 "mustaches" 127 }
+			gene_being={ "no_being" 127 "no_being" 127 }
+			wings={ "no_wings" 0 "no_wings" 0 }
+			claws={ "no_claws" 0 "no_claws" 0 }
+			hairstyles={ "scripted_character_hairstyles_01" 244 "all_hairstyles" 0 }
+			beards={ "rtt_beards" 104 "no_beard" 0 }
+		}
+	}
+	enabled=yes
+}
+
+### Shadowsong ###
+
+# Maiev Shadowsong
+maiev_shadowsong_dna = { # by Runtd
```

Comment:
Her facial features are based on her reforged design while her hair, skin and marks colors are based on her unmasked NPC
![image](https://user-images.githubusercontent.com/75175790/132961552-3cd2b318-b938-4b59-b7e8-36992437ce84.png)
![image](https://user-images.githubusercontent.com/75175790/132961539-d53386a2-50da-445c-876b-07f733d8a7d7.png)

Reply:
I would say her chin looks flat, it's more defined on the art.

Reply:
Pretty much all of her chin-related genes are maxed, so I changed her jaw instead, this is the best I could make her look.
![image](https://user-images.githubusercontent.com/75175790/133885740-45d1deaf-6a28-466c-980d-7aaa15cda2b7.png)
This may also result in her childen (especially her sons) to have a big jaw, like this :
![image](https://user-images.githubusercontent.com/75175790/133885545-986e3023-6af5-4460-a4e4-69503ec54d84.png)
Knowing this, should I push this change anyway ?

Reply:
> Pretty much all of her chin-related genes are maxed, so I changed her jaw instead, this is the best I could make her look.

I would say it's too big.
Increase Jaw Height and Jaw Angle a bit.

Reply:
Modified her jaw as requested and pushed the changes, please test it and tell me if you're ok with the new dna.